decidim-core/app/views/layouts/decidim/shared/_layout_center.html.erb

```diff
@@ -1,8 +1,8 @@
-<% columns ||= 6 %>
 <% main_enabled = true unless local_assigns.has_key?(:main_enabled) %>
 <% main_tag = redesign_enabled? && main_enabled ? :main : :div %>
-<%= content_tag main_tag, class: "layout-1col" do %>
-  <div class="<%= columns == 10 ? "cols-10" : columns == 8 ? "cols-8" : "cols-6" %>">
-    <%= yield %>
-  </div>
+<% columns ||= 6 %>
+<% css_class = columns == 10 ? "cols-10" : columns == 8 ? "cols-8" : "cols-6" %>
```

Comment:
@entantoencuanto could you help Hugo to refactor this and extract to a helper

Reply:
After reviewing it again, I would like to mark this comment as unresolved and suggest a change here.

Just changing this line and using a case statement instead of multiple ternary operators would make this code clearer.

Reply:
Done! Please review again

Reply:
Reviewed. Feel free to merge it!